Scientific_workflows/FractionalCoverOfWaterProject/Manual_classification_workflow.ipynb

```diff
@@ -0,0 +1,994 @@
+{
```

Comment:
<p>What is drone_res_tgt and why can't I change it?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GeoscienceAustralia/dea-notebooks/pull/663/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
<p>Done.</p>